src/ruby/puppet-server-lib/puppet/server/puppet_config.rb

```diff
@@ -53,5 +53,15 @@ def self.initialize_puppet(puppet_config)
 
     Puppet::ApplicationSupport.configure_indirector_routes("master")
 
+    oid_defns = Puppet::SSL::Oids.parse_custom_oid_file(Puppet[:trusted_oid_mapping_file])
+    if oid_defns.nil?
```

Comment:
This would be more idiomatic ruby if the conditional was reversed and the if were just `if oid_defns`.